imap_processing/codice/l0/decom_codice.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+from imap_processing import imap_module_directory
+from imap_processing.decom import decom_packets
+
+if __name__ == "__main__":
+    # Define paths
+    packet_file = Path("housekeeping_data.bin")
+    xtce_document = Path(
+        f"{imap_module_directory}/codice/packet_definitions/P_COD_NHK.xml"
+    )
+
+    # Decommutated packets
+    decomposed_packets = decom_packets(packet_file, xtce_document)
+
+    # Print decommutated packets
+    for packet in decomposed_packets:
+        if packet.header["PKT_APID"].raw_value == 1136:
+            print(packet.data)
```

Comment:
You could do similar to what I have done for SWE and SWAPI.
```suggestion
from imap_processing import decom, imap_module_directory


def decom_packets(packet_file: str):
    """Decom CoDICE data packets using CoDICE packet definition.

    Parameters
    ----------
    packet_file : str
        Path to data packet path with filename.

    Returns
    -------
    List
        List of all the unpacked data
    """
    # TODO: how to call other packet's definition? May be, you could pass another optional input that passes apid and based on apid, you can grab correct packet definition?
    xtce_document = (
       f"{imap_module_directory}/codice/packet_definitions/P_COD_NHK.xml"
    )
    return decom.decom_packets(packet_file, xtce_document)
```